lambda_functions/transform_lambda.py

```diff
@@ -50,7 +60,13 @@ def lambda_handler(event, context):
                     "data": encoded_output,
                 }
                 output_records.append(output_record)
-
+            else:
```

Comment:
This `else` seems misaligned? It needs to be indented if it is for the `if` directly above this block

Also, why do we need the `else`? Why do we want to add to `output_records` if there were no metric results?

Reply:
So The firehose gives these records to lambda to process. If we don't process them than it keeps record of what we didn't give a response for and stores them in the S3 bucket as process errors, this makes it so that we tell the firehose we  dropped these and we are okay with that.

Reply:
it shows correctly indented on my screen. will poke around